bazel/thirdparty/openssl.BUILD

```diff
@@ -77,18 +79,17 @@ filegroup(
 )
 
 select_file(
-    name = "openssl_exe_file",
+    name = "openssl_data",
```

Comment:
Should we just select_file the configuration file?

Reply:
The issue here was that the config file is not being declared as an explicit output of the openssl target.
And I couldn't find a way to make the `configure_make` rule to specify out_data files. Only out_data folders.